### PR TITLE
feat m: [Crypto/DD] Add Coinglass Liquidations

### PR DESCRIPTION
**Pull request by jfarid27**
# Description

- Adds crypto liquidations data from Coinglass to the dd menu for the loaded coin.
- Adds tests for Coinglass model's existing functions. Previously tests were empty.

## Issue

#2400

## Screenshots

Figure_1

- [x] Summary of the change / bug fix.
- [x] Link # issue, if applicable.
- [x] Screenshot of the feature or the bug before/after fix, if applicable.
- [x] Relevant motivation and context. 
- [] ~List any dependencies that are required for this change.~

# How has this been tested?

Added new test suites for coinglass_model and ran VCR tests.

* Please describe the tests that you ran to verify your changes. 
* Provide instructions so we can reproduce. 
* Please also list any relevant details for your test configuration.

# Checklist:

- [x] Update [our Hugo documentation](https://openbb-finance.github.io/OpenBBTerminal/) following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/website).
- [x] Update our tests following [these guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/tests).
- [x] Make sure you are following our [CONTRIBUTING guidelines](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/CONTRIBUTING.md).
- [x] If a feature was added make sure to add it to the corresponding [scripts file](https://github.com/OpenBB-finance/OpenBBTerminal/tree/main/scripts).

# Others
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My code passes all the checks pylint, flake8, black, ... To speed up development you should run `pre-commit install`.
- [x] New and existing unit tests pass locally with my changes. You can test this locally using `pytest tests/...`.

### CODE REVIEW

**Review by jose-donato — APPROVED** [review]

File: website/content/terminal/crypto/dd/liquidations/_index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+```
+usage: liquidations
+            [--export {csv,json,xlsx}] [-h]
+```
+
+Display the liquidation data for a certain cryptocurrency. [Source: https://coinglass.github.io/API-Reference/]
```
Comment (jfarid27) [inline website/content/terminal/crypto/dd/liquidations/_index.md:6]: Is this documentation ok? I would definitely request a review here to make sure it makes sense.

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py [hunk 2]
```diff
@@ -56,7 +56,16 @@ def check_cg_id(symbol: str):
 class DueDiligenceController(CryptoBaseController):
     """Due Diligence Controller class"""
 
-    CHOICES_COMMANDS = ["load", "fundrate", "oi", "active", "change", "nonzero", "eb"]
+    CHOICES_COMMANDS = [
+        "load",
+        "fundrate",
+        "oi",
+        "liquidations",
```
Comment (jfarid27) [inline openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:63]: Is there a required ordering here?

**Comment by jose-donato** [comment]
Thank you for the contribution @jfarid27!! Looking good 🚀

File: openbb_terminal/cryptocurrency/due_diligence/dd_controller.py (hunk 2 above)
Comment (jose-donato) [reply openbb_terminal/cryptocurrency/due_diligence/dd_controller.py:63]: nop

File: website/content/terminal/crypto/dd/liquidations/_index.md (hunk 1 above)
Comment (jose-donato) [reply website/content/terminal/crypto/dd/liquidations/_index.md:6]: Yea it's fine, if you want to make it better you can add the image you uploaded there as a sneak peek

https://user-images.githubusercontent.com/1673206/186202157-31d1f6df-5f08-4849-98df-abb17c513fe1.png

**Pull request merged**